src/components/Combat/combatUtils.ts

```diff
@@ -1,16 +1,23 @@
 import { requestCombatRegister } from '@src/services/combat.services';
 import { iPosition } from '@src/types/mapInterfaces';
-import { iRequestCombatRegister } from '@src/types/requestInterfaces';
+import { useToastAlert } from '@src/hooks/useToastAlert';
 
 export const getCombatToken = async (
   userPos: iPosition,
   gymId: string
 ): Promise<string | null> => {
   // try to register combat
-  const data: iRequestCombatRegister | null = await requestCombatRegister(
-    userPos,
-    gymId
-  );
+  const [data, error] = await requestCombatRegister(userPos, gymId);
+
+  // code 400
+  if (error == 400) {
```

Comment:
@Woynert There are another errors that can be 400, for example gyms without loomies or if the user attack recently the gym.

Reply:
@SilviaPabon Of course!